test/verify/check-docker-storage

```diff
@@ -65,7 +66,7 @@ def initially_loopbacked(machine):
 # device is added.
 #
 def initially_without_vgroup(machine):
-    return machine.image in [ "fedora-24", "rhel-7" ]
```

Comment:
Should we really be removing rhel-7 from this, since this about a new image shouldn't this be left untouched?